packages/os/configure-snapshotter.service

```diff
@@ -0,0 +1,28 @@
+[Unit]
+Description=Configure Snapshotter
+Before=containerd.service
+After=settings-applier.service bootstrap-commands.service
+Requires=settings-applier.service bootstrap-commands.service
```

Comment:
You don't need these dependencies since you're running in `multi-user.target`; they'll be completed by the time this unit is queued to start.